src/Uno.UI/Uno.UI.csproj

```diff
@@ -174,6 +174,10 @@
 		</ProjectReference>
 	</ItemGroup>
 
+	<ItemGroup>
+	  <UpToDateCheckInput Remove="UI\Xaml\Controls\ScrollViewer\ScrollViewerViewChangingEventArgs.cs" />
+	</ItemGroup>
```

Comment:
This was auto-generated by Visual Studio. I'm not sure if this is needed.

Reply:
Yes, VS often generates these, you can undo this change